src/components/forms/validator.ts

```diff
@@ -59,20 +61,27 @@ export const ok = (): undefined => undefined
 
 export const mustBeEthereumAddress = memoize(
   (address: string): ValidatorReturnType => {
+    const disabledFeatures = getNetworkConfigDisabledFeatures()
+    const isENSEnabled = disabledFeatures.every((feature) => feature !== FEATURES.ENS)
```

Comment:
When checking that at least 1 element meet the condition I prefer to use `some` function, as it stops checking once it finds 1 element. When using `some` remember that you have to check equality `===`

Also as this same functionality is defined inside two functions in the same file, does it make sense to create an internal function so they share the logic?

Reply:
:tada: I wasn't thinking clearly with the `disabledFeatures` vs `isENSEnabled`, thanks!

Reply:
Changed the `every` to `some` and also created a utility function in the config file, so it's used on every place it's required. [`9732d08` (#1570)](https://github.com/gnosis/safe-react/pull/1570/commits/9732d087420cc4dff5d54ecaaf985dc284d4e9a1)